decidim-meetings/spec/system/explore_meetings_spec.rb

```diff
@@ -568,19 +566,17 @@
       end
     end
 
-    it_behaves_like "has drawer attachments" do
+    it_behaves_like "has redesigned attachments" do
       let(:attached_to) { meeting }
     end
 
     shared_examples_for "a closing report page" do
       it "shows the closing report" do
-        skip_unless_redesign_enabled
-
         visit_component
         click_link translated(meeting.title)
         expect(page).to have_i18n_content(meeting.closing_report)
 
-        within "#drawer_content_frame" do
+        within "[data-content]" do
```

Comment:
Can we use here a class or other css selector ?

Reply:
Explanation [here](https://github.com/decidim/decidim/pull/10329#discussion_r1100123951)